client/entitlements.go

```diff
@@ -6,11 +6,7 @@ import (
 	"github.com/replicatedhq/replicated/pkg/types"
 )
 
-func (c *Client) CreateEntitlementSpec(name string, spec string, appID string) (*types.EntitlementSpec, error) {
-	appType, err := c.GetAppType(appID)
-	if err != nil {
-		return nil, err
-	}
+func (c *Client) CreateEntitlementSpec(name string, appType string, spec string, appID string) (*types.EntitlementSpec, error) {
```

Comment:
It's a little weird that every other function starts with appID and AppType as the first two parameters, and this one starts with the 'name' parameter.  Updating this for consistency would be a good idea, unless there's a strong reason otherwise.